neurons/validator.py

```diff
@@ -52,6 +52,11 @@ def __init__(self):
         super(Validator, self).__init__()
         load_dotenv()
 
+        # temperary penalty for a naughty miner
```

Comment:
lol. let's make sure tho revert this after it served its purpose

Reply:
sure！